assets/docs/rishit-dagli/mirnet-tfjs/1.md

```diff
@@ -0,0 +1,27 @@
+# Placeholder rishit-dagli/mirnet-tfjs/1
```

Comment:
This line should be
`# Module rishit-dagli/mirnet-tfjs/1`
since there is an actual saved model to be published.

Reply:
Thanks for pointing this out, this has now been fixed.